recipe/meta.yaml

```diff
@@ -271,20 +276,33 @@ outputs:
         # decide on the cuda variant
         # https://github.com/conda-forge/tensorflow-feedstock/issues/162
         - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
-        - bazel 4.2.2
+        - bazel >=5.2.0
+        - libabseil
+        - grpc-cpp
+        - numpy 1.20.*  # [py<310]
+        - numpy 1.21.*  # [py>=310]
+        - openssl
+        - libprotobuf
+        - {{ pin_subpackage('tensorflow-base', exact=True) }}
       host:
         - python
         - pip
+        - numpy 1.20.*  # [py<310]
+        - numpy 1.21.*  # [py>=310]
         - packaging
         - setuptools
         - wheel
+        - libabseil
+        - grpc-cpp
+        - libprotobuf
```

Comment:
Did estimator start depending on this, or are we just making a previous dependence explicit?